src/test/regress/README.md

```diff
@@ -81,6 +81,35 @@ Adding a new test file is quite simple:
 4. Check that the output is as expected
 5. Copy the `.out` file from `results` to `expected`
 
+### Adding an alternative test output
+
+Sometimes the test output changes because we run tests in different configurations.
+The most common example is an output that changes in different Postgres versions.
+If we don't care about the change, we try to add [a normalization rule](https://github.com/citusdata/citus/blob/main/ci/README.md#normalize_expectedsh), or find
```

Comment:
I think `care` could be subjective word here. I have a strong opinion on avoiding the additional test outputs, because they are always pain. If we don't strongly reject those, it is very easy to have tons of such tests. 

So, I'd probably re-write this in a way that: `Alternative outputs are highly discouraged .. only use when strictly necessary .. you can first try to change the test such that you still test what you want but the output is different..   then you can try using some utility functions in the tests like coordinator_plan, and be creative to add more such tests if you need.. then you can try normalization .. then you can even consider dropping the specific test lines if the test is not very critical in the sense that we have other tests checking the same code paths`

Reply:
also, maybe mention reducing the test verbosity via: set verbosity terse, or client_min_messages TO ERROR or others